opal/mca/base/mca_base_pvar.h

```diff
@@ -319,7 +319,7 @@ OPAL_DECLSPEC int mca_base_pvar_register (const char *project, const char *frame
  * associated with a component.
  *
  * While quite similar to mca_base_pvar_register(), there is one key
- * difference: pvars registered this this function will automatically
+ * difference: pvars registered with this function will automatically
```

Comment:
This is a nice catch but I would create a different PR just for this typo as this has nothing to do with SPC. Ask @bosilca .

Reply:
Technically @thananon is right. Practically, you will not be able to push a 1 line commit without approval from the devels. So leave it here ...

Reply:
I take it I shouldn't update the copyright information on this file...?